lib/debug/deprecate.js

```diff
@@ -80,9 +80,7 @@ function deprecate(description, condition, options) {
 
   let message = formatMessage(description, options);
 
-  warn(message);
-  warn('');
-  warn(getStackTrace());
+  warn(`${message}\n\n${getStackTrace()}`);
```

Comment:
Noticed during testing that it's better to use `console.warn` only once per deprecation.